omc3/scripts/luminosity_imbalance.py

```diff
@@ -0,0 +1,187 @@
+from math import sqrt
+from functools import reduce
+from operator import mul
+import pathlib
+import re
+
+import tfs
+from tfs.tools import significant_digits
+
+from omc3.utils.logging_tools import get_logger
+from omc3.definitions.constants import PLANES
+from generic_parser import entrypoint, EntryPointParameters
+
+from omc3.run_kmod import LSA_FILE_NAME as LSA_RESULTS
+from omc3.run_kmod import RESULTS_FILE_NAME as RESULTS
+from omc3.run_kmod import EXT
+
+LOG = get_logger(__name__)
+
+BETASTAR = 'BETSTAR'
+ERR = 'ERR'
+LABEL = 'LABEL'  # Column containing the IP/Beam names
+BEAMS = ('B1', 'B2')
+IPS = ('ip1', 'ip5')
+LABELS = [f'{i}{b}' for i in IPS for b in BEAMS]
+
+
+def _validate_for_imbalance(df):
```

Comment:
Validation should rather not return anything and simply raise in case it detects an issue so we save indentation levels in scripts. @JoschD @lmalina ?

Reply:
The imbalance is only computed if all 4 files are presented, the script stills merges if that's not the case

Reply:
As Mael said. If it confuses rename to '_all_data_for_imbalance_present()'